util/topgen/rust.py

```diff
@@ -214,6 +180,9 @@ def __init__(self, top_info, name_to_block: Dict[str, IpBlock], version_stamp: D
         self.regwidth = int(top_info["datawidth"])
         self.file_header = RustFileHeader(version_stamp)
 
+        # TODO: Don't hardcode the address space used for software.
+        self.addr_space = "hart"
```

Comment:
Maybe this should be a constructor parameter?

Reply:
This will already be fixed in a follow-up PR: https://github.com/lowRISC/opentitan/pull/23122 For now, I would hold-off changing that in this PR.